File: Python/Estrutura repetitiva/fatorial.py
# Variável
fat = 1

# Entrada de dado
n = int(input("Digite um valor entre 0 a 15: "))

# Estrutura repetitiva
for i in range(n, 0, -1): # É necessário colocar o -1 para ele fazer a conta de trás para frente!
    fat = fat * i

# Saída de dados
print(f"O fatorial de {n}! = {fat}")

File: C#/Estrutura repetitiva/fatorial.cs
// Variável
int fat, n, i;

fat = 1;

// Entrada de dados
Console.Write("Digite um valor entre 0 a 15: ");
n = int.Parse(Console.ReadLine());

// Estrutura repetitiva
for (i = n; i > 0; i--) // O i-- serve para fazer a operação ao contrário
{
    // Algoritmo matemático
    fat *= i;
}

// Saída de dados
Console.WriteLine("O fatorial de " + n + "!" + " = " + fat);